netfields/rest_framework.py

```diff
@@ -7,6 +7,11 @@
 
 from netfields.compat import text_type
 from netfields.mac import mac_unix_common
+from netfields.fields import (
+    InetAddressField as _DBInetAddressField,
+    CidrAddressField as _DBCidrAddressField,
+    MACAddressField as _DBMACAddressField
+)
```

Comment:
Would doing this as simply

```
import netfields.fields
```
 be a little more obvious?

Reply:
Maybe it could be even simplier this way:

```
from netfields import fields

[...]
NetModelSerializer.serializer_field_mapping[fields.InetAddressField] = InetAddressField
```